Sources/SKCore/BuildSystemManager.swift

```diff
@@ -72,6 +72,22 @@ public actor BuildSystemManager {
   }
 }
 
+/// Splits `message` on newline characters such that each chunk is at most `maxChunkSize` bytes long.
```

Comment:
This doesn't quite seem accurate to me since chunks can exceed `maxChunkSize` bytes for lines in the input that already exceeded that limit. Or should we be splitting long lines too?

Reply:
Yeah, I know… It’s good enough for a `fileprivate` function for command line argument splitting though. I just added a comment now.

```
This will only split along newline boundary. If a single line is longer than `maxChunkSize`, it won't be split. This is fine for compiler argument splitting since a single argument is rarely longer than 800 characters.
```